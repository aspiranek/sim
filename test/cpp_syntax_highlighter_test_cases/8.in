char trolololo_char = '\n';
char char_trolololo = '\\';
char trolololo_char_ = trolololo_char + char_trolololo;


