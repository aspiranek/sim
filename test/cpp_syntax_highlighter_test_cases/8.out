<table class="code-view"><tbody><tr><td id="L1" line="1"></td><td><span style="color:#0000ff;font-weight:bold">char</span> trolololo_char <span style="color:#515125">=</span> <span style="color:#e0a000">&apos;<span style="color:#d923e9">\n</span>&apos;</span><span style="color:#515125">;</span>
</td></tr><tr><td id="L2" line="2"></td><td><span style="color:#0000ff;font-weight:bold">char</span> char_trolololo <span style="color:#515125">=</span> <span style="color:#e0a000">&apos;<span style="color:#d923e9">\\</span>&apos;</span><span style="color:#515125">;</span>
</td></tr><tr><td id="L3" line="3"></td><td><span style="color:#0000ff;font-weight:bold">char</span> trolololo_char_ <span style="color:#515125">=</span> trolololo_char <span style="color:#515125">+</span> char_trolololo<span style="color:#515125">;</span>
</td></tr><tr><td id="L4" line="4"></td><td>
</td></tr><tr><td id="L5" line="5"></td><td>
</td></tr><tr><td id="L6" line="6"></td><td></td></tr></tbody></table>